src/services/MilestoneService.js

```diff
@@ -457,10 +457,16 @@ class MilestoneService {
 
       afterMined(!milestone.projectId, `${etherScanUrl}tx/${txHash}`, milestoneId);
     } catch (err) {
-      const message = `Something went wrong with the Milestone ${
-        milestone.projectId > 0 ? 'update' : 'creation'
-      }. Is your wallet unlocked? ${etherScanUrl}tx/${txHash} => ${JSON.stringify(err, null, 2)}`;
-      ErrorHandler(err, message);
+      const showMessageInPopup = err.data && err.data.showMessageInPopup;
```

Comment:
showMessageInPopup is a front-end property, should be handled another way discussed with @dehghani-giveth